### PR TITLE
Strengthen part of username and password creation process

### LINKED ISSUE #2090: Strengthen part of username and password creation process
This file is called when a user begins setting up OpenEMR for the first time: https://github.com/openemr/openemr/blob/master/setup.php

Make the following happen via HTML5 attributes:
- initial user password min length of 12
- database  password min length of 12

Also:
- the "initial user" of "admin" is too easy to guess
- write a PHP function (or Javascript function, can be on the front end that is fine) that generates 3 random letters  + "-admin-"  + 2 random numbers
- e.g: HLV-admin-87
- then take this functions output and fill in the HTML for "initial user"
- This will make bruteforce attacks against the login page approximately 1,700,000 harder at no real cost to the user (50%  less extra characters than SSN or Phone Number)
- make the username a minimum of 12 characters as well to ensure if they don't go with our randomized suggestion they still pick something with some complexity (enforce with html5)

### PR DESCRIPTION
**Pull request by alpha-gamma**
Fixes  #2090

initial user password min length of 12 
database password min length of 12 
generate random user name 
make the username a minimum of 12 characters
Updated field's help text as well

@bradymiller Please review and check for any mistakes or improvements

### CODE REVIEW

File: setup.php [hunk 1]
```diff
@@ -262,7 +262,7 @@ function cloneClicked() {
...
 <TR VALIGN='TOP'><TD><span class='text'>Database Name: </span></TD><TD><INPUT TYPE='TEXT' VALUE='openemr' NAME='dbname' SIZE='30'></TD><TD><span class='text'>(This is the name of the OpenEMR database in MySQL - 'openemr' is the recommended)</span><br></TD></TR>
 <TR VALIGN='TOP'><TD><span class='text'>Login Name: </span></TD><TD><INPUT TYPE='TEXT' VALUE='openemr' NAME='login' SIZE='30'></TD><TD><span class='text'>(This is the name of the OpenEMR login name in MySQL - 'openemr' is the recommended)</span><br></TD></TR>
-<TR VALIGN='TOP'><TD><span class='text'>Password: </span></TD><TD><INPUT TYPE='PASSWORD' VALUE='' NAME='pass' SIZE='30'></TD><TD><span class='text'>(This is the Login Password for when PHP accesses MySQL - it should be at least 8 characters long and composed of both numbers and letters)</span><br></TD></TR>\n";
+<TR VALIGN='TOP'><TD><span class='text'>Password: </span></TD><TD><INPUT TYPE='PASSWORD' VALUE='' NAME='pass' SIZE='30' minlength=12></TD><TD><span class='text'>(This is the Login Password for when PHP accesses MySQL - it should be at least 12 characters long and composed of both numbers and letters)</span><br></TD></TR>\n";
```
Comment (bradymiller) [inline setup.php]: put singlequotes around the 12 in minlength attribute

File: setup.php [hunk 2]
```diff
@@ -356,10 +356,11 @@ function cloneClicked() {
...
-<TR VALIGN='TOP' class='noclone'><TD><span class='text'>Initial User Password:</span></TD><TD><INPUT SIZE='30' TYPE='PASSWORD' NAME='iuserpass' VALUE=''></TD><TD><span class='text'>(This is the password for the initial user account above.)</span></TD></TR>
+            echo "<TR VALIGN='TOP' class='noclone'><TD><span class='text'>Initial User:</span></TD><TD><INPUT SIZE='30' TYPE='TEXT' NAME='iuser' VALUE='".$randomusername."' minlength=12></TD><TD><span class='text'>(This is the login name of user that will be created for you. Limit this to one word with at least 12 characters.)</span></TD></TR>
+<TR VALIGN='TOP' class='noclone'><TD><span class='text'>Initial User Password:</span></TD><TD><INPUT SIZE='30' TYPE='PASSWORD' NAME='iuserpass' VALUE='' minlength=12></TD><TD><span class='text'>(This is the password for the initial user account above - it should be at least 12 characters long.)</span></TD></TR>
```
Comment (bradymiller) [inline setup.php]: for both lines above, put singlequotes around the 12 in minlength attribute

Comment (bradymiller) [reply setup.php]: Lets instruct that the Initial User Password should be composed of letters and numbers

Comment (alpha-gamma) [reply setup.php]: But, as of now there is no validation on password for that.

**Comment by alpha-gamma** [comment]
I guess we should make these field as required as well to stop the form submission without the passwords.

**Comment by bradymiller** [comment]
Agree on the required

**Comment by alpha-gamma** [comment]
Cool. I will make the changes.

**Comment by bradymiller** [comment]
Code looks good and will bring into codebase now. Thanks for the contribution!

**Pull request merged**